src/explorers/ethereum/etherscan.ts

```diff
@@ -1,32 +1,50 @@
 import request from '../../services/request';
 import { stripHashPrefix } from '../../utils/stripHashPrefix';
 import { buildTransactionServiceUrl } from '../../services/transaction-apis';
-import { BLOCKCHAINS, isTestChain } from '../../constants/blockchains';
+import { BLOCKCHAINS } from '../../constants/blockchains';
 import { TransactionData } from '../../models/transactionData';
 import { TRANSACTION_APIS, TRANSACTION_ID_PLACEHOLDER } from '../../constants/api';
-import { ExplorerAPI, ExplorerURLs, IParsingFunctionAPI } from '../../models/explorers';
+import { ExplorerAPI, IParsingFunctionAPI } from '../../models/explorers';
 import CONFIG from '../../constants/config';
 import { SupportedChains } from '../../constants/supported-chains';
 
 const MAIN_API_BASE_URL = 'https://api.etherscan.io/api?module=proxy';
-const TEST_API_BASE_URL = 'https://api-ropsten.etherscan.io/api?module=proxy';
-const serviceURL: ExplorerURLs = {
-  main: `${MAIN_API_BASE_URL}&action=eth_getTransactionByHash&txhash=${TRANSACTION_ID_PLACEHOLDER}`,
-  test: `${TEST_API_BASE_URL}&action=eth_getTransactionByHash&txhash=${TRANSACTION_ID_PLACEHOLDER}`
-};
+
+function getApiBaseURL (chain: SupportedChains): string {
+  switch (chain) {
+    case SupportedChains.Ethmain:
+      return MAIN_API_BASE_URL;
+    case SupportedChains.Ethropst:
+      return 'https://api-ropsten.etherscan.io/api?module=proxy';
+    case SupportedChains.Ethrinkeby:
+      return 'https://api-rinkeby.etherscan.io/api?module=proxy';
+    case SupportedChains.Ethgoerli:
+      return 'https://api-goerli.etherscan.io/api?module=proxy';
+    case SupportedChains.Ethsepolia:
+      return 'https://api-sepolia.etherscan.io/api?module=proxy';
```

Comment:
I like this is clear but I think we could use a map?

ie:
```
return 'https://api-${key[SupportedChains.Ethsepolia]}.etherscan.io/api?module=proxy'
```

Reply:
It sounds good! I'll change it.